noir-projects/aztec-nr/aztec/src/history/contract_inclusion/test.nr

```diff
@@ -0,0 +1,35 @@
+// TODO (12225): the #[aztec] macro currently does not support being used in aztec-nr, hence we cannot do any variation of the below.
+// Replace the hacky work around in noir-contracts/counter_contract which tests the contract inclusion.
```

Comment:
Why can't it be used here? Is it because it expects to be in a contract crate?

And is it expected to ever be the case? If not then this should be nuked.